core/Cargo.toml

```diff
@@ -355,7 +355,7 @@ prometheus-client = { version = "0.22.2", optional = true }
 tracing = { version = "0.1", optional = true }
 # for layers-dtrace
 probe = { version = "0.5.1", optional = true }
-crc32c = "0.6.5"
+crc32c = { git = "https://github.com/zowens/crc32c", rev = "04eabbc" }
```

Comment:
Also need to be optional behind `service-s3` feature.

Reply:
Fixed by https://github.com/apache/opendal/pull/4643

Reply:
Cool! I'll rebase this patch once upstream released.